### ISSUE TITLE
dbc.Input has different behaviour to dcc.Input when value is outside min/max

### ISSUE THREAD

**Issue by samlishak-artemis** [facultyai/dash-bootstrap-components#621]
- dash version: `1.20.0`
- dash-bootstrap-components version: `0.12.2`
- components affected by bug: `dbc.Input`

### What is happening?

When the `value` of a `dbc.Input` is outside the range of `min`/`max`, the number is still returned.

### What should be happening?

To match behaviour of `dcc.Input`, the component value must be None in this situation. From the Dash docs:

> _Fixed and enhanced in Dash v1.1.0_
> 
> Number type is now close to native HTML5 `input` behavior across browsers. We also apply a strict number casting in callbacks: valid number converts into corresponding number types, and invalid number converts into None. E.g. `dcc.Input(id='range', type='number', min=2, max=10, step=1)` typing 3 and 11 will return respectively integer three and None in Python callbacks.

### Code

In the example below, typing 11 will return 11 instead of None.

```python
import dash
import dash_core_components as dcc
import dash_bootstrap_components as dbc
import dash_html_components as html
from dash.dependencies import Input, Output, State, MATCH, ALL

app = dash.Dash(__name__, suppress_callback_exceptions=True)

app.layout = html.Div(
    [
        dbc.Input(id="range", type="number", min=2, max=10, step=1),
        html.Div(id="value", children=""),
    ]
)


@app.callback(
    Output("value", "children"),
    Input("range", "value"),
)
def show_value(value):
    return str(value)


if __name__ == "__main__":
    app.run_server(debug=True)

```

**Comment by yunlongxu-artemis** [comment]
Is there a way to fix this issue in this PR?

if not, please create a ticket in the PostPOC epic with label `technical-debt` to make sure we catch this.

**Comment by tcbegley** [comment]
Thanks for raising @samlishak-artemis!

Will work on a fix for the next release.

**Comment by tcbegley** [comment]
This is resolved in version `0.13.0`. Please update dash-bootstrap-components and let me know if you have any further problems

```sh
pip install -U dash-bootstrap-components
```
